StreamChatSwiftUI.xcodeproj/project.pbxproj

```diff
@@ -502,6 +502,25 @@
 		A3600B43283F664E00E1C930 /* StartPage.swift in Sources */ = {isa = PBXBuildFile; fileRef = A3600B42283F664E00E1C930 /* StartPage.swift */; };
 		A3828EAD283F6CFE00538258 /* StartPage.swift in Sources */ = {isa = PBXBuildFile; fileRef = A3828EAC283F6CFE00538258 /* StartPage.swift */; };
 		A3D7B0DF2840E23100E308B3 /* UIView+AccessibilityIdentifier.swift in Sources */ = {isa = PBXBuildFile; fileRef = A3D7B0DE2840E23100E308B3 /* UIView+AccessibilityIdentifier.swift */; };
+		AD2DDA5D2CB033460040B8D4 /* ChatThreadList.swift in Sources */ = {isa = PBXBuildFile; fileRef = AD2DDA5C2CB033460040B8D4 /* ChatThreadList.swift */; };
```

Comment:
weird, I can't open the project because of: 
```
The project ‘StreamChatSwiftUI’ is damaged and cannot be opened due to a parse error. Examine the project file for invalid edits or unresolved source control conflicts.
```

Reply:
Ah, I have some stuff not pushed yet, maybe it is because of that, I'll fix this tomorrow 👍

Reply:
Still having an issue here - on both Xcode 15 and 16. Develop works fine though.

Reply:
Really? Weird... I can option it without any issue 🤔

Reply:
It seems that on CI there is also no issue 🤔

Reply:
Did you pulled the changes?